src/css/reviewPage.css

```diff
@@ -0,0 +1,18 @@
+/* Review Page */
+.review-page__page-heading__text {
+  color: var(--color-primary-darker);
+}
+
+.review-page__page-heading__link {
+  text-align: right;
+  margin-top: 1.8em;
+}
+
+.review-page__page-info__label-text {
+  color: var(--color-gray);
+}
+
+.review-page__page-info__value-text {
+  color: var(--color-gray-dark);
```

Comment:
`vads-u-color--gray` and might be able to use `vads-u-font-size--md` if you could get away 17px instead of 18px: https://design.va.gov/foundation/utilities/font-size